backend/app/main/views.py

```diff
@@ -30,3 +31,24 @@ def get_all_mentors():
             m = Mentor(name=name, email=email)
             list_of_mentors.append(m.serialize())
     return jsonify(list_of_mentors)
+
+
+# get all clients from Airtable
+@main.route("/clients", methods=["GET"])
+def get_all_clients():
+    response = requests.get(
+        "https://api.airtable.com/v0/appw4RRMDig1g2PFI/Clients",
+        headers={"Authorization": str(os.environ.get("API_KEY"))},
+    )
+    response_json = response.json()
+
+    list_of_clients = []
+    for r in response_json["records"]:
+        print(r["fields"])
```

Comment:
We can remove this so we don't have leftover print statements (this should have been removed from the Mentors endpoint as well)

Reply:
Thanks, got it!